SkyFloatingLabelTextField/SkyFloatingLabelTextFieldExample/Example0/ShowcaseExampleViewController.swift

```diff
@@ -263,7 +263,7 @@ class ShowcaseExampleViewController: UIViewController, UITextFieldDelegate {
             return
         }
 
-        if email.characters.isEmpty {
```

Comment:
So is this something that was part of string in Swift prior to 4.2 but now isn't necessary?

Reply:
It's still there, but it's being deprecated

Reply:
Noice